.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "esbonio.sphinx.confDir": ""
+}
```

Comment:
nit: add space

Reply:
Oh yeah I need to remove this .vscode. This is some scruff. I'll update the .gitignore